src/components/new-request/submit-request/reserve-submit.vue

```diff
@@ -128,30 +127,32 @@ export default class ReserveSubmitButton extends Vue {
 
     let data: any
     let request: NameRequestI
-    const requestAction = this.getRequestActionOriginal || this.getRequestActionCd
```

Comment:
"getRequestActionOriginal" was obsolete (never set) so I deleted it. That simplified this assignment, so I deleted it and moved "getRequestActionCd" into the calls where it is used, below.